app/controllers/concerns/vet360/transactionable.rb

```diff
@@ -22,7 +23,11 @@ def check_transaction_status!
     private
 
     def service
-      VAProfile::ContactInformation::Service.new @current_user
+      if Flipper.enabled?(:va_v3_contact_information_service, @current_user)
```

Comment:
Do you want to name this feature toggle `va_v2_contact_information_service` instead of v3 here?

Reply:
This is already a feature flipper in prod. I can change it if you want. The flipper will probably be removed after testing anyways though.